### PR TITLE
PR 7

### PR DESCRIPTION
**Pull request by OksanaEfanova**

### CODE REVIEW

**Review by TestGitHub100500 — CHANGES_REQUESTED** [review]
456

**Review by TestGitHub100500 — APPROVED** [review]

**Review by TestGitHub100500 — DISMISSED** [review]

**Review by TestGitHub100500 — CHANGES_REQUESTED** [review]
daaa

**Comment by OksanaEfanova** [comment]
comment12

**Comment by OksanaEfanova** [comment]
comment13

**Review by TestGitHub100500 — APPROVED** [review]

**Review by TestGitHub100500 — CHANGES_REQUESTED** [review]
NOO

File: test100500_for slack [hunk 1]
```diff
@@ -0,0 +1 @@
+test100500_for slack
```
Comment (TestGitHub100500) [inline test100500_for slack:1]: comment

**Review by TestGitHub100500 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**